RELEASE.md

```diff
@@ -0,0 +1,4 @@
+Release type: patch
+
+
+Added pyupgrade to pre-commit hooks. Also ran it and made sure there are no (unexpected) test breakages after the changes.
```

Comment:
I will just write:

This release adds pyupgrade to pre-commit

Reply:
done, thanks!